src/Components/OATPropertyEditor/PropertyList.tsx

```diff
@@ -221,6 +247,100 @@ export const PropertyList = ({
         handleSelectorPosition(e);
     };
 
+    const listItemOnClick = (item) => {
+        setCurrentPropertyIndex(item.index);
+        setModalOpen(true);
+        setModalBody(FormBody.property);
+    };
```

Comment:
Could we define this funciton inside of `getListItems`?

I.e. so we don't need to pass it as a parameter?